framework/global/modularity/imodulesetup.h

```diff
@@ -2,7 +2,7 @@
 //  MuseScore
 //  Music Composition & Notation
 //
-//  Copyright (C) 2019 MuseScore BVBA and others
+//  Copyright (C) 2020 MuseScore BVBA and others
```

Comment:
Rather "2019, 2020"

Reply:
De facto this is a new file, with the old one it has only the header and file name.